### DISCUSSION TITLE
Questions about QSO count in the exported map and in image from wavelog wlimager.

### DISCUSSION THREAD

**Discussion by BG6HJE-edu** [wavelog/wavelog#1171] [Q&A] [answered]
The QSOs in the Americas on both sides of the exported map are different, which results in fewer QSOs in the Americas in the static output generated by wavelog wlimager. Is it a bug or should I set up something? TNX DE BG6HJE, 73!

Exported map from https://images.wavelog.org/img/map?call=bg6hje :
![image](https://github.com/user-attachments/assets/a53026d9-19e2-4ca6-9fd5-789566f6e311)

Image generated by wavelog wlimager:
![image](https://github.com/user-attachments/assets/0845d530-99e2-47bd-8244-40a3d570a033)

**Comment by HB9HIL** [comment, marked as answer]
The WLimager will be replaced pretty soon. Hang tight about that.

**Reply by BG6HJE-edu** [reply]
Thanks!
